core/controllers/base.py

```diff
@@ -143,9 +142,9 @@ def __init__(self, request, response):  # pylint: disable=super-init-not-called
         # Initializes the return dict for the handlers.
         self.values = {}
 
-        self.user = current_user_services.get_current_user()
+        self.current_user = current_user_services.get_current_user()
```

Comment:
Would it be possible to not define current_user on self, if it's not needed? Perhaps we should just have current_user_services.get_current_user_id() in place of these two lines, too.